source/module_base/tool_check.cpp

```diff
@@ -77,7 +77,7 @@ void CHECK_STRING(std::ifstream &ifs,const std::string &v,bool quit)
 		{
 			std::cout << " read in value = " << v_in << std::endl;
```

Comment:
Shall we use something like `std::cerr` here? @dyzheng

Reply:
In my opinion, there is no need to use std::cerr, only fatal memory error should use that for printing error message immediately.